java/iot3/core/build.gradle

```diff
@@ -20,10 +20,7 @@ repositories {
 }
 
 dependencies {
-    // This dependency is exported to consumers, that is to say found on their compile classpath.
-    //api 'com.hivemq:hivemq-mqtt-client:1.3.0'
-
-    // This dependency is used internally, and not exposed to consumers on their own compile classpath.
+    // Internal dependencies, not exposed to consumers on their own compile classpath.
     implementation 'com.hivemq:hivemq-mqtt-client:1.3.0'
```

Comment:
(Premier commentaire).
Revue de code après avoir tout récupérer dans IntellIJ dans un projet Java.
Le plugin intégré Checkmarx voit des vulnérabilités dans les dépendances du client mqtt en version 1.3.0.
Voir le lien: https://mvnrepository.com/artifact/com.hivemq/hivemq-mqtt-client/1.3.0.
Une version 1.3.1 corrige +sieurs vulné mais il en resterait deux...

Reply:
Updating hiveMQ MQTT client to version 1.3.3, not much to do for the remaining vulnerabilities until further updates from HiveMQ.